Scoop/Supporting/Base.lproj/LaunchScreen.storyboard

```diff
@@ -11,15 +14,27 @@
             <objects>
                 <viewController id="01J-lp-oVM" sceneMemberID="viewController">
                     <view key="view" contentMode="scaleToFill" id="Ze5-6b-2t3">
-                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                        <rect key="frame" x="0.0" y="0.0" width="393" height="852"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
-                        <color key="backgroundColor" xcode11CocoaTouchSystemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                        <subviews>
+                            <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" fixedFrame="YES" image="launchScreen" translatesAutoresizingMaskIntoConstraints="NO" id="uxD-Xn-qpV">
+                                <rect key="frame" x="0.0" y="0.0" width="393" height="852"/>
+                                <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                            </imageView>
+                        </subviews>
                         <viewLayoutGuide key="safeArea" id="6Tk-OE-BBY"/>
+                        <color key="backgroundColor" systemColor="systemBackgroundColor"/>
                     </view>
                 </viewController>
                 <placeholder placeholderIdentifier="IBFirstResponder" id="iYj-Kq-Ea1" userLabel="First Responder" sceneMemberID="firstResponder"/>
             </objects>
-            <point key="canvasLocation" x="53" y="375"/>
+            <point key="canvasLocation" x="52.307692307692307" y="374.64454976303318"/>
         </scene>
     </scenes>
+    <resources>
+        <image name="launchScreen" width="375" height="812"/>
+        <systemColor name="systemBackgroundColor">
+            <color white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+        </systemColor>
+    </resources>
 </document>
```

Comment:
Do you have any constraints for the launch screen images? Check on a different screen size.

Reply:
<img width="214" alt="image" src="https://user-images.githubusercontent.com/100049615/231227844-78c8943b-f68a-43b0-a85d-817d4fef1379.png">
<img width="233" alt="image" src="https://user-images.githubusercontent.com/100049615/231227890-f5286ee9-73b1-4644-a1ce-deb33e2060e4.png">
<img width="216" alt="image" src="https://user-images.githubusercontent.com/100049615/231227927-7c8d3062-4f85-4dd5-a429-bac8c8f808d4.png">

I adjusted some storyboard settings, seems okay? Do you know if this is how you test storyboard sizing

Reply:
ye looks good.

Reply:
ok ty vin dow 🤲